src/argilla/client/feedback/integrations/huggingface/card/argilla_template.md

```diff
@@ -75,7 +75,7 @@ The **questions** are the questions that will be asked to the annotators. They c
 
 | Question Name | Title | Type | Required | Description | Values/Labels |
 | ------------- | ----- | ---- | -------- | ----------- | ------------- |
-{% for question in argilla_questions %}| {{ question.name }} | {{ question.title }} | {{ question.__class__.__name__  }} | {{ question.required | default(true, true) }} | {{ question.description | default("N/A", true) }} | {% if question.settings.type == 'rating' %}{{ question.settings.options | map(attribute="value") | list }}{% else %} N/A {% endif %} |
+{% for question in argilla_questions %}| {{ question.name }} | {{ question.title }} | {{ question.__class__.__name__  }} | {{ question.required | default(true, true) }} | {{ question.description | default("N/A", true) }} | {% if question.settings.type in ["rating", "label_selection", "multi_label_selection", "ranking"] %}{{ question.settings.options | map(attribute="value") | list }}{% else %}N/A{% endif %} |
```

Comment:
I think it would be good to have a constant in the client with all the available kind of questions and use that constant everywhere, so when a new question is added we just need to add a string in one place.

Reply:
Yes I agree, for most of the stuff we're already re-using the pre-defined types and just looping over those to avoid re-writing those, but in this case we're using Jinja2 to generate the card, and the behavior here needs to be manually checked, as not all the questions are the same under this scenario.